resources/elk6/logstash-vulnwhisperer-template.json

```diff
@@ -0,0 +1,73 @@
+{
+  "order": 0,
+  "index_patterns": "logstash-vulnwhisperer-*",
+  "mappings": {
+    "doc": {
+      "properties": {
+        "plugin_id": {
+          "type": "integer"

```

Comment:
this was the value that was bringing issues and needed to be changed to float right? just to make sure before I do a change that is not supposed to be done :D

Reply:
I'd be inclined to actually make this a keyword, we can't guarantee all sources will have integer only plugin_ids and it's not likely something people want to numerically sort on?

Reply:
you are right, I agree with you change it to a key :)